webui/react/src/stores/determinedInfo.tsx

```diff
@@ -23,78 +16,21 @@ export const initInfo: DeterminedInfo = {
   version: process.env.VERSION || '',
 };
 
-const DeterminedInfoContext = createContext<DeterminedInfoContext>({
-  info: NotLoaded,
-  updateInfo: noOp,
-});
-
-export const DeterminedInfoProvider: React.FC<PropsWithChildren> = ({ children }) => {
-  const [info, setInfo] = useState<Loadable<DeterminedInfo>>(NotLoaded);
-  return (
-    <DeterminedInfoContext.Provider value={{ info, updateInfo: setInfo }}>
-      {children}
-    </DeterminedInfoContext.Provider>
-  );
-};
-
-export const useFetchInfo = (canceler: AbortController): (() => Promise<void>) => {
-  const context = useContext(DeterminedInfoContext);
-  if (context === null) {
-    throw new Error('Attempted to use useDeterminedInfo outside of Determinednfo Context');
+const info = observable<Loadable<DeterminedInfo>>(NotLoaded);
```

Comment:
Should we guard this with 'private'?

Reply:
that's for class fields, right? with this approach, i think just not exporting the value directly and only exporting the `readOnly()` value should suffice, unless there's an issue i'm missing